Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/RunSummary.cs

```
@@ -10,28 +10,30 @@ namespace CosmosBenchmark
 
     public class RunSummary
     {
-        public string pk { get; set; } 
+        public string pk => this.BenchmarkConfig?.ResultsPartitionKeyValue;
```

nit: Can `BenchmarkConfig` be null?...we will always has some default configs.

---

It can be null because it's not passed in the CTOR. If someone does new RunSummary().pk it will throw a null ref instead of just returning null.

---

Fixed in latest iteration